devops/scripts/load_and_validate_env.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+# set -o xtrace
+#
+# Usage:
+#    load_and_validate_env.sh
+#
+
+if [ ! -f "config.yaml" ]; then
+  if [ -z "${USE_ENV_VARS_NOT_FILES:-}" ]; then
+    echo -e "\e[31m»»» 💥 Unable to find config.yaml file, please create file and try again!\e[0m"
+    #exit
+  fi
+else
+    # Validate no duplicate keys in config
+    has_dupes=$(yq e '... comments=""|.. | select(. == "*") | {(path | .[-1]): .}| keys' config.yaml | uniq -d)
+    if [ -n "${has_dupes:-}" ]; then
+      echo -e "\e[31m»»» 💥 There are duplicate keys in your config, please fix and try again!\e[0m"
+      exit
+    fi
+
+    # Validate config schema
+    pajv validate -s config_schema.json -d config.yaml
+
+    # declare the variable and export to the caller's context
+    STRIP_COMMENTS="... comments=\"\""
```

Comment:
this is very similar to the code in another script
is there a way to package this up so that we don't duplicated this code?